lib/matplotlib/tests/test_backend_svg.py

```diff
@@ -145,3 +144,71 @@ def test_svgnone_with_data_coordinates():
         buf = fd.read().decode()
 
     assert expected in buf
+
+
+def test_gid():
+    """Test that object gid appears in output svg."""
+    from matplotlib.offsetbox import OffsetBox
+    from matplotlib.axis import Tick
+
+    fig = plt.figure()
+
+    ax1 = fig.add_subplot(131)
+    ax1.imshow([[1., 2.], [2., 3.]], aspect="auto")
+    ax1.scatter([1, 2, 3], [1, 2, 3], label="myscatter")
+    ax1.plot([2, 3, 1], label="myplot")
+    ax1.legend()
+    ax1a = ax1.twinx()
+    ax1a.bar([1, 2, 3], [1, 2, 3])
+
+    ax2 = fig.add_subplot(132, projection="polar")
+    ax2.plot([0, 1.5, 3], [1, 2, 3])
+
+    ax3 = fig.add_subplot(133, projection="3d")
+    ax3.plot([1, 2], [1, 2], [1, 2])
+
+    fig.canvas.draw()
```

Comment:
Do you need the draw() here?

Reply:
If I don't draw it, the ticks will not be the ones that are exported, because the ticker only creates them when drawing the figure.

Reply:
ah, of course.